src/sharing/components/Recipient.jsx

```diff
@@ -225,17 +219,6 @@ RecipientPlusX.contextTypes = {
   t: PropTypes.func.isRequired
 }
 
-export const ContactSuggestion = ({ contact }) => {
-  const name = getPrimaryEmail(contact)
-  const url = getPrimaryCozy(contact)
-  return (
-    <div className={styles['recipient']}>
-      <Avatar text={getInitiales(name)} size={'small'} textId={name} />
-      <Identity name={name} url={url} />
-    </div>
-  )
-}
-
-const getInitiales = name => {
+export const getInitials = name => {
```

Comment:
Should we move this method out of this file? You have already started to lighten the file ;)

Reply:
We should create a method in cozy-doctype/contacts and use it. Same case for : 
- getPrimaryEmail and getPrimaryCozy